tests/test_benchmarks/test_zhu2019.py

```diff
@@ -0,0 +1,69 @@
+from pathlib import Path
+
+import pytest
+from pytest import approx
+
+from brainio.assemblies import BehavioralAssembly
+from brainscore import benchmark_pool
+from tests.test_benchmarks import PrecomputedFeatures
+
+
+@pytest.mark.private_access
+class TestZhu2019:
+
+    @pytest.mark.parametrize('benchmark', [
+        'Zhu2019-response_match',
+        'Zhu2019-accuracy',
+    ])
+    def test_in_pool(self, benchmark):
+        assert benchmark in benchmark_pool
+
+    @pytest.mark.parametrize('benchmark, expected_ceiling', [
+        ('Zhu2019-response_match', approx(0.8957, abs=0.0001)),
+        ('Zhu2019-accuracy', 1),
+    ])
+    def test_benchmark_ceiling(self, benchmark, expected_ceiling):
+        benchmark = benchmark_pool[benchmark]
+        ceiling = benchmark._ceiling(benchmark._assembly)
+        if "accuracy" in benchmark.identifier:
+            assert ceiling.sel(aggregation='center') == expected_ceiling
+        else:
+            assert ceiling == expected_ceiling
```

Comment:
```suggestion
        assert ceiling.sel(aggregation='center') == expected_ceiling
```
there should be no distinction here, all ceilings are formatted the same